### PR TITLE
Mobile layout - report overview

### PR DESCRIPTION
**Pull request by SharonV33**
image

components added 
**Atoms**
- ReportType

**Molecules**
- MultibleReportTypes
- ReportExtraOptions

**Organisms**
- SingleReport

**Templates**
- AllReports

**Routes**
- reports

### CODE REVIEW

**Review by johancvb — CHANGES_REQUESTED** [review]
Unnecessary new-lines in Home.svelte (line 42), Multiple in stead of Multible

File: src/components/organisms/SingleReport.svelte [hunk 1]
```diff
@@ -1,32 +1,68 @@
...
-  <p>Was iemand die mij aanrandde tijdens het dansen. Hij bleef maar doorgaan ondanks dat ik al een aantal keer zei dat hij moest stoppen. Lees meer...</p>
+  <p>
+    Was iemand die mij aanrandde tijdens het dansen. Hij bleef maar doorgaan ondanks dat ik al
+    een aantal keer zei dat hij moest stoppen. <a href="www.google.com"> Lees meer...</a>
+  </p>
+
+  <section class="extraInfo">
+    <ReportExtraOptions/>
+    <section>
+      <p>23:00 | 21/5/2021</p>
```
Comment (veerleprins) [inline src/components/organisms/SingleReport.svelte:64]: Glad you made this static! Maybe you can just make this a variable with the static data and put that in here 😄

File: src/components/templates/AllReports.svelte [hunk 2]
```diff
@@ -19,28 +18,46 @@
 <style lang="scss">
   // Importing fonts, vars, etc.
   @import 'src/styles/index.scss';
+
   * {
-    font-family: $standard-font;
+    font-family: 'Open Sans';
```
Comment (veerleprins) [inline src/components/templates/AllReports.svelte:23]: I'll take a look at how to load the font-family in a more global way, but I'll come back to this later 😄 👍🏼

**Review by johancvb — APPROVED** [review]
Good job! (⌐■_■)

**Review by veerleprins — APPROVED** [review]
Nice job Sharon 👍🏼

**Pull request merged**